src/components/TransactionModal/TransactionModal.tsx

```diff
@@ -124,8 +128,22 @@ export const TransactionModal: React.FC<TransactionModalProps> = ({ status, onCl
       </StyledTransactionIllustration>
       <Dialog
         title={stepDetails?.title}
+        primaryButton={
+          status === ExtrinsicStatus.VoucherSizeLimitExceeded
+            ? {
+                text: 'Go to discord',
+                to: 'https://discord.com/invite/DE9UN3YpRP',
```

Comment:
Let's extract this to a constant in `config/urls`. Note that we already have `JOYSTREAM_DISCORD_URL` in there - this is a general Discord link. For this usage I'd like to direct users directly to the right channel so they don't need to ask around. I'm coordinating with the community on which channel this should be, will provide you with a link at a later time